Build-Wheels.py

```diff
@@ -0,0 +1,66 @@
+import json
+
+import requests
+from packaging.requirements import InvalidRequirement
+from packaging.requirements import Requirement
+
+# Global variables for branches and constraints files for the branches to be downloaded
+# (length of each list should be the same and so the positions for branch and it's specific constraints file)
+IDF_BRANCH = ['release/v5.0', 'release/v5.1', 'master']
+IDF_CONSTRAINTS = ['v5.0', 'v5.1', 'v5.2']
+
+# TODO temporary solution
+install_optional = True
+
+
+idf_resources_url = 'https://raw.githubusercontent.com/espressif/esp-idf/'
+requirements_txt = []
+requirements_set = set()
+
+# Download all requirements from all the branches requirements and constraints files
+for i in range(len(IDF_BRANCH)):
```

Comment:
This is the Pythonic and faster way to get the index while iterating through the list:
```python
for i, item in enumerate(IDF_BRANCH):
```